src/ser/mod.rs

```diff
@@ -39,12 +40,7 @@ impl From<u8> for Error {
     }
 }
 
-#[cfg(feature = "std")]
-impl ::std::error::Error for Error {
-    fn description(&self) -> &str {
-        ""
-    }
-}
+impl serde::ser::StdError for Error {}
```

Comment:
Ditto on adding a feature guard

Reply:
Not needed. https://docs.rs/serde/1.0.147/src/serde/ser/mod.rs.html#118-123